finance/proposal-calls/cycle4/pickering_specreduce.md

```diff
@@ -0,0 +1,28 @@
+### Title
+
+Development, Coordination, and Community Engagement involving specreduce and other spectroscopy-related tools by Timothy Pickering
+
+### Project Team
+Timothy Pickering
+
+### Project Description
+This is to continue specreduce development and outreach that was carried out in previous cycles.
+
+### Project / Work
+Some specific areas that we expect to tackle during the coming cycle include:
+
+* Generic manual wavelength calibration classes (i.e. analogs to "identify" from iraf, should use previous specreduce tools for this where possible)
+* Automatic wavelength calibration using existing templates (i.e. "reidentify", or "full template" from pypeit)
+* Improved WCS tools and expanded examples of their use within spectroscopic contexts
+* Support for / workflows with multi-object spectrographs
+* Support for fiber-based spectrographs, both single and multi-object as well as fiber-based IFUs
```

Comment:
Will these result in PRs to specutils, specreduce, both, or none?

Reply:
> Will these result in PRs to specutils, specreduce, both, or none?

`specreduce` definitely, `specutils` possibly if it would make the most sense in specific cases. `gwcs` is another possibility if more attention is given to its spectroscopy-related functionality.